docker-compose.yml

```diff
@@ -70,7 +67,7 @@ services:
     volumes:
       - shared-volume:/opt/snapshot-sender/data
     command: >-
-      --logging.level.root=DEBUG
+      --logging.level.root=INFO
```

Comment:
can this come from env var?

Reply:
Not really this is a property picked up by spring's logging framework  and this is how it is provided

Reply:
The main point of this change was to reduce all the noise in the circleci console.

Reply:
ah ok, I'd at least create a variable in the docker compose file so that you are only defining the log level the once

Reply:
Ok done.